### PR TITLE
Jailhouse+buildroot

### PR DESCRIPTION
**Pull request by paresh-bhagat12**
10.1 doc updates for jailhouse and buildroot

### CODE REVIEW

**Review by jeevantelukula — DISMISSED** [review]

File: source/linux/Foundational_Components/Hypervisor/Jailhouse.rst [hunk 1]
```diff
@@ -427,14 +427,14 @@ the demo are:
 
     .. code-block:: console
 
-        root@am62xx-evm: jailhouse cell load k3-am625-sk-linux-demo /usr/libexec/jailhouse/linux-loader.bin -a 0x0 -s "kernel=0xe0200000 dtb=0xe0000000" -a 0x1000 /boot/Image -a 0xe0200000 /boot/tisdk-jailhouse-inmate-am62xx-evm.rootfs.cpio -a 0xe28d4000 /usr/share/jailhouse/inmate-k3-am625-sk.dtb -a 0xe0000000
+        root@am62xx-evm: jailhouse cell load k3-am625-sk-linux-demo /usr/libexec/jailhouse/linux-loader.bin -a 0x0 -s "kernel=0xe0200000 dtb=0xe0000000" -a 0x1000 /boot/Image -a 0xe0200000 /boot/tisdk-jailhouse-inmate-am62xx-evm.rootfs.cpio -a 0xe2a5e000 /usr/share/jailhouse/inmate-k3-am625-sk.dtb -a 0xe0000000
```
Comment (cshilwant) [inline source/linux/Foundational_Components/Hypervisor/Jailhouse.rst:430]:
```suggestion
        root@<machine>: jailhouse cell load k3-am625-sk-linux-demo /usr/libexec/jailhouse/linux-loader.bin -a 0x0 -s "kernel=0xe0200000 dtb=0xe0000000" -a 0x1000 /boot/Image -a 0xe0200000 /boot/tisdk-jailhouse-inmate-am62xx-evm.rootfs.cpio -a 0xe2a5e000 /usr/share/jailhouse/inmate-k3-am625-sk.dtb -a 0xe0000000
```

**Review by cshilwant — CHANGES_REQUESTED** [review]
In commit message title of 67cccd150d12bb6800cb401f59de501577ae1b0f,

s/Buildoot/Buildroot

File: source/buildroot/Building_Buildroot_Image.rst
Comment (cshilwant) [inline source/buildroot/Building_Buildroot_Image.rst:1]: In this entire file, for directives like code-block, note, etc code sections/content should clearly line up with directive - https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace

**Review by cshilwant — CHANGES_REQUESTED** [review]

File: source/linux/Foundational_Components/Hypervisor/Jailhouse.rst (hunk 1 above)
Comment (StaticRocket) [reply source/linux/Foundational_Components/Hypervisor/Jailhouse.rst:430]: @cshilwant , why would you want to make this block generic? It's literally calling `k3-am625-sk-linux-demo` with device tree specific information.

File: source/buildroot/Building_Buildroot_Image.rst
Comment (StaticRocket) [reply source/buildroot/Building_Buildroot_Image.rst:1]: I would like to see this file reworked at some point but I don't know if we should push for it here. Feels a little unfair to jam a doc refactor into a +16,-14 PR.

Comment (paresh-bhagat12) [reply source/buildroot/Building_Buildroot_Image.rst:1]: @cshilwant @StaticRocket Noted. I will have to update other docs too. I will do it after the first sync for 10.1 release

**Review by cshilwant — DISMISSED** [review]

File: source/linux/Foundational_Components/Hypervisor/Jailhouse.rst [hunk 2]
```diff
@@ -399,7 +393,7 @@ the demo are:
 
     .. code-block:: console
 
-        root@am62xx-evm: jailhouse enable /usr/share/jailhouse/cells/k3-am625-sk.cell
+        root@<machine>: jailhouse enable /usr/share/jailhouse/cells/k3-am625-sk.cell
```
Comment (StaticRocket) [inline source/linux/Foundational_Components/Hypervisor/Jailhouse.rst:396]: A handful of these blocks are inherently machine specific. I do not think that context should be removed.

**Review by StaticRocket — CHANGES_REQUESTED** [review]

File: source/linux/Foundational_Components/Hypervisor/Jailhouse.rst (hunk 2 above)
Comment (cshilwant) [reply source/linux/Foundational_Components/Hypervisor/Jailhouse.rst:396]: @StaticRocket Well that was intentional. Apparently AM62X includes am62xxsip-evm, am62xx-evm and am62xx-lp-evm. That's the reason @paresh-bhagat12 generalized it as machine as jailhouse is currently supported on both am62xx-evm and am62xx-lp-evm

Comment (StaticRocket) [reply source/linux/Foundational_Components/Hypervisor/Jailhouse.rst:396]: Those devices don't have a different prompt though, do they?

Comment (cshilwant) [reply source/linux/Foundational_Components/Hypervisor/Jailhouse.rst:396]: @StaticRocket it's they have different prompt :wink: - https://git.yoctoproject.org/meta-ti/tree/meta-ti-bsp/conf/machine?h=scarthgap

**Review by jsuhaas22 — DISMISSED** [review]

**Comment by cshilwant** [comment]
\+ @jeevantelukula for reviews

File: source/linux/Foundational_Components/Hypervisor/Jailhouse.rst [hunk 3]
```diff
@@ -427,14 +421,14 @@ the demo are:
 
     .. code-block:: console
 
-        root@am62xx-evm: jailhouse cell load k3-am625-sk-linux-demo /usr/libexec/jailhouse/linux-loader.bin -a 0x0 -s "kernel=0xe0200000 dtb=0xe0000000" -a 0x1000 /boot/Image -a 0xe0200000 /boot/tisdk-jailhouse-inmate-am62xx-evm.rootfs.cpio -a 0xe28d4000 /usr/share/jailhouse/inmate-k3-am625-sk.dtb -a 0xe0000000
+        root@<machine>: jailhouse cell load k3-am625-sk-linux-demo /usr/libexec/jailhouse/linux-loader.bin -a 0x0 -s "kernel=0xe0200000 dtb=0xe0000000" -a 0x1000 /boot/Image -a 0xe0200000 /boot/tisdk-jailhouse-inmate-am62xx-evm.rootfs.cpio -a 0xe2a5e000 /usr/share/jailhouse/inmate-k3-am625-sk.dtb -a 0xe0000000
```
Comment (AashvijShenai) [inline source/linux/Foundational_Components/Hypervisor/Jailhouse.rst:424]: @paresh-bhagat12 @cshilwant , I notice that the addresses are different. Any plausible way to highlight that it has changed? I'm wondering in the niche case of someone who has the previous commands but wants the latest images

Comment (paresh-bhagat12) [reply source/linux/Foundational_Components/Hypervisor/Jailhouse.rst:424]: @AashvijShenai These addresses are generated via jailhouse cell tools for linux demo according to inmate device tree, cpio etc so they change in every release ( also we moved to 6.6.58). They might change if user uses a different kernel or cpio. We already have a script in our jailhouse image which runs the linux demo. So user does not need to run the above command themselves, but run the script instead. This section just explains a bit what script is doing.

**Review by jeevantelukula — APPROVED** [review]

File: source/linux/Foundational_Components/Hypervisor/Jailhouse.rst (hunk 3 above)
Comment (AashvijShenai) [reply source/linux/Foundational_Components/Hypervisor/Jailhouse.rst:424]: Thanks

**Review by AashvijShenai — APPROVED** [review]

**Review by StaticRocket — APPROVED** [review]

**Comment by StaticRocket** [comment]
@paresh-bhagat12 please resolve conflicts in: `source/linux/Foundational_Components/Hypervisor/Jailhouse.rst`

**Comment by praneethbajjuri** [comment]
> @paresh-bhagat12 please resolve conflicts in: `source/linux/Foundational_Components/Hypervisor/Jailhouse.rst`

@paresh-bhagat12  conflicts still there.

**Review by cshilwant — APPROVED** [review]

**Review by jeevantelukula — APPROVED** [review]

**Review by jsuhaas22 — APPROVED** [review]

**Pull request merged**